tests/test_docker.py

```
@@ -0,0 +1,206 @@
+
+
+class TestDockerActions:
+    async def test_list(self, _mock_graphql: AsyncMock) -> None:
+        _mock_graphql.return_value = {
+            "docker": {"containers": [{"id": "c1", "names": ["plex"], "state": "running"}]}
+        }
+        tool_fn = _make_tool()
+        result = await tool_fn(action="list")
+        assert len(result["containers"]) == 1
+
+    async def test_start_container(self, _mock_graphql: AsyncMock) -> None:
```

Ruff (0.15.0)

[warning] Fixture `_mock_graphql` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)